proto/tendermint/liquidity/v1beta1/query.proto

```diff
@@ -65,6 +65,38 @@ service Query {
         };
     }
 
+    // Get specific liquidity pool corresponding to the pool_coin_denom.
+    rpc LiquidityPoolByPoolCoinDenom (QueryLiquidityPoolByPoolCoinDenomRequest) returns (QueryLiquidityPoolResponse) {
+        option (google.api.http).get = "/tendermint/liquidity/v1beta1/pools/{pool_coin_denom}";
```

Comment:
Wouldn't it conflict with current endpoint for querying pool by pool id?
https://github.com/tendermint/liquidity/blob/cdc771781e4e02f9ad03d3691f3b465847f8c700/proto/tendermint/liquidity/v1beta1/query.proto#L38